packages/gatsby-theme/src/blocks/ContactBlock/index.js

```diff
@@ -4,8 +4,8 @@ import { Contact } from "ui"
 
 const ContactBlock = ({ heading, align, contactInfo }) => {
   return (
-    <Contact>
-      <Contact.Header align={align}>{heading}</Contact.Header>
+    <Contact align={align}>
+      {heading}
```

Comment:
![image](https://user-images.githubusercontent.com/41605725/107533848-f9754780-6b84-11eb-8075-d92aeb77f32f.png)
The heading is a bit too close to the components below. I think some spacing was lost with this change.